settings.gradle

```diff
@@ -25,4 +25,5 @@ include 'eventmesh-test'
 include 'eventmesh-spi'
 include 'eventmesh-connector-plugin:eventmesh-connector-api'
 include 'eventmesh-connector-plugin:eventmesh-connector-rocketmq'
+include 'eventmesh-rest'
```

Comment:
Maybe it is better to rename this module to eventmesh-rest-api like other spi module if you hope to add another implementations in the future.

Reply:
> Maybe it is better to split this module into two modules like `eventmesh-openschema-api` and `eventmesh-openschema-rest` like other spi module if you hope to add another implementations in the future.

That's a good suggestion. Should I create a new ```eventmesh-openschema-plugin``` module under main```EventMesh``` project, and put ```eventmesh-openschema-api``` and ```eventmesh-openschema-rest``` in it. Or create ```eventmesh-openschema-plugin``` module under ```eventmesh-rest```. I prefer the former one, but #435 suggests all RESTful APIs in ```eventmesh-rest```, how do you think?

Reply:
I think the first one is good, since the module is main about openschema

Reply:
I think @ruanwenjun 's idea of creating eventmesh-openschema-rest under its own plugin is good. Basically, the plugin becomes very self-contained.. Regarding #435, I will update to suggest creating a new module called eventmesh-admin-plugin which contains its own eventmesh-admin-rest along with other dependencies.